custom/icds/messaging/indicators.py

```diff
@@ -406,7 +405,7 @@ def get_report_fixture(self, report_id):
     def clear_caches(self):
```

Comment:
This doesn't appear to be called anywhere, including in the PR that added it, https://github.com/dimagi/commcare-hq/pull/15357/

Reply:
Does any place make sense to call this to invalidate the cache?

Reply:
Not sure, probably ideally it'd be when the config changes and also when new data comes in, which seems quite a bit harder to detect.